### ISSUE TITLE
RTC Schedule Wakeup not working (?)

### ISSUE THREAD

**Issue by hamdroid** [JeffCurless/argoneon#59]
After installing the required CR1220 and adding an RTC schedule using the provided scripts, make the argon eon shutdown according to schedule.

The turn on schedule is not honored, forcing me to hard start the eon again.

Is the wakeup routine confirmed working? Am I missing something? 

Tested with latest. Thanks for input; kinda stuck.

**Comment by JeffCurless** [comment]
I have made *zero* changes to the RTC code, and I have *never* been able to have it wake up the system.  I'll leave this here as an outstanding issue to see if it can be fixed at sometime.

I'm sure we are doing something wrong...  just no sure what.

**Comment by hamdroid** [comment]
Hey Jeff, I was writing primarily to revive the issue, I saw you made no changes to these features. I was hopeful I may be doing something wrong and you may have an idea what to do. I also tried deactivating the RTC service and loading in the RTC myself; but the code to interact with the alarms is not something I fully understood in my short review :) maybe someone out there reads this and has some insight :)

**Comment by hamdroid** [comment]
Didn't mean to close :D

**Comment by JeffCurless** [comment]
Yeah, I was reviewing the code and I don't see how it actually works... obviously it really does not...

**Comment by hamdroid** [comment]
Good news :) I kept experimenting and actually got the box to suspend itself and wake back up :)

And it's a relative easy fix to do:

[argoneon](https://github.com/JeffCurless/argoneon/tree/main)
/argoneond.py line 1003

 os.system("shutdown now -h") --> os.system("halt")

Reading the linux man page -h is equivalent to poweroff. This may also work with shutdown -H  which also means halt but I didn't test that yet.

Anyhow. Told this system to shutdown and start backup 5min later via service. Pinged it during the duration; was "dead" and then came back :)

If you are interested I can submit a merge request.

Cheers!

**Comment by JeffCurless** [comment]
Opening to fix bug

**Comment by JeffCurless** [comment]
Thanks, making the change (and testing shutdown now -H).

**Comment by B08Z** [comment]
I am still have an issue where the EON does not power up. 
I have used Shutdown now -H and Halt both are the same. Pi shuts down but does not wake up.

**Comment by JeffCurless** [comment]
Can you push your changes in?  I have attempted your code change either I am doing something wrong in testing, or something else is going bad,,, I cannot get the RTC wakeups/shutdown to work properly.

**Comment by donjalex2** [comment]
Hello, I recently setup my Argon Eon case. All is working good with the exception of RTC… it shows the correct date and time after configuration, but it won’t power on with the schedule I set. I do have a battery installed. For example, I scheduled it for Tuesday and Thursday to turn on at 6:30 AM. No luck…
